### ISSUE TITLE
ipywidget error on minimal install

### ISSUE THREAD

**Issue by jennirinker** [dask/distributed#3020]
This issue was filed/closed in [dask-jobqueue](https://github.com/dask/dask-jobqueue/issues/271) but doesn't seem to be open here. I just ran into it today, figured I'd file a bug report.

I have a minimal install of dask (just dask and dask.distributed) on miniconda. A cell with the following code

```python
from dask.distributed import LocalCluster
cluster = LocalCluster()
cluster
```

throws the following traceback.
```python
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
~\AppData\Local\Continuum\miniconda3\envs\dask-tutorial\lib\site-packages\IPython\core\formatters.py in __call__(self, obj)
    916             method = get_real_method(obj, self.print_method)
    917             if method is not None:
--> 918                 method()
    919                 return True
    920 

~\AppData\Local\Continuum\miniconda3\envs\dask-tutorial\lib\site-packages\distributed\deploy\cluster.py in _ipython_display_(self, **kwargs)
    206 
    207     def _ipython_display_(self, **kwargs):
--> 208         return self._widget()._ipython_display_(**kwargs)

~\AppData\Local\Continuum\miniconda3\envs\dask-tutorial\lib\site-packages\distributed\deploy\cluster.py in _widget(self)
    143             pass
    144 
--> 145         from ipywidgets import Layout, VBox, HBox, IntText, Button, HTML, Accordion
    146 
    147         layout = Layout(width='150px')

ModuleNotFoundError: No module named 'ipywidgets'
```
Let me know if you need more configuration info.

**Comment by mrocklin** [comment]
@jcrist if memory serves you have a decent solution of this in dask-yarn.  Are you still ok to push that work upstream when you get back from vacation?

**Comment by jcrist** [comment]
Sure, I'll make a PR tomorrow.

**Comment by jcrist** [comment]
Should be fixed by #3023.
